.ci.yaml

```diff
@@ -2687,6 +2687,7 @@ targets:
           {"dependency": "xcode", "version": "14a5294e"},
           {"dependency": "gems", "version": "v3.3.14"}
         ]
+      cpu: arm64
```

Comment:
This will not be checked it, it's just to kick off and validate this test now passed on an arm host.